express/storage/memory_storage.js

```diff
@@ -1,47 +1,2 @@
-'use strict';
-
-const BaseStorage = require("./base_storage");
-
-class MemoryStorage extends BaseStorage {
-    constructor(prefixKey) {
-        super(prefixKey);
-        this.__data = {};
-    }
-
-    async get(key) {
-        return this.__data[this.prefixKey + key];
-    }
-
-    async set(key, value) {
-        this.__data[this.prefixKey + key] = value;
-    }
-
-    async setex(key, value, ttl) {
-        //TODO: add support for ttl
-        this.__data[this.prefixKey + key] = value;
-    }
-
-    async del(key) {
-        delete this.__data[this.prefixKey + key];
-    }
-
-    async hget(key, hashKey) {
-        let hashMap = this.__data[this.prefixKey + key];
-        if(hashMap) {
-            return hashMap[hashKey];
-        }
-        return null;
-    }
-
-    async hset(key, hashKey, value) {
-        let hashMap = this.__data[this.prefixKey + key] || {};
-        hashMap[hashKey] = value;
-        this.__data[this.prefixKey + key] = hashMap;
-    }
-
-    async hgetall(key) {
-        return this.__data[this.prefixKey + key];
-    }
-}
-
+const MemoryStorage = require('../../storage/memory_storage');
```

Comment:
Framework specific import is necessity here ? @jinalvirani 
Check if we can avoid this

Reply:
yes, necessary for backward compatibility as old extension code has import statement from '/express/redisstorage'